prometheus_speedtest/prometheus_speedtest.py

```diff
@@ -16,13 +16,14 @@
 
 flags.DEFINE_string('address', '0.0.0.0', 'address to listen on')
 flags.DEFINE_integer('port', 9516, 'port to listen on')
+flags.DEFINE_multi_integer('servers', [], 'speedtest server(s) to use - leave empty for auto-selection')
```

Comment:
We should use `DEFINE_list` instead of `DEFINE_multi_integer`:

With multi_integer you have to specify the flag multiple times if you want multiple values: `--servers=0 --servers=1 --servers=2`. This is clunkier than using a list which you can specify as just `--servers=0,1,2`.